### DISCUSSION TITLE
Runtime Error / Hydration failed

### DISCUSSION THREAD

**Discussion by MarenOelixtown** [neuefische/questions#849] [Web] [answered]
Hi all, I created a form for adding new Plants in our app-project and I am at the point to check wit console.log(); witch data arrives after submitting. But i can't because the attached Error Message appears...maybe anybody has a hint how to proceed? Many thank for help in advance!
https://github.com/MarenOelixtown/plant-care-app/commit/3873f0e4c050d268a66e2d73f609442a1a4e3b9c

![Screenshot 2024-07-03 133437](https://github.com/neuefische/questions/assets/152329875/7ad95470-cb09-4299-a452-212f729b927f)

**Comment by NikRoe** [comment, marked as answer]
Hi there,

the issue you are currently facing is related to how you structure your HTML.

In `/pages/createplant/index.js` you return JSX that is wrapped by a styled component `FormPageContainer` which is a form element.
Amongst other things, this JSX also contains the component `CreatPlantForm` which is also a form element.

The error comes from wrapping a form inside of a form.

I recommend you change the `FormPageContainer` to be a div instead of a form. This should get rid of the error.

**Reply by MarenOelixtown** [reply]
Thanks a lot, very helpful hint!!!
